web/apps/labelstudio-e2e/src/support/commands.ts

```diff
@@ -17,9 +17,9 @@ declare namespace Cypress {
 }
 //
 // -- This is a parent command --
-Cypress.Commands.add('login', (email, password) => {
-  console.log('Custom command example: Login', email, password);
-});
+// Cypress.Commands.add('login', (email, password) => {
+//   console.log('Custom command example: Login', email, password);
+// });
 //
```

Comment:
are we intending to put this back in at some point? we should probably just delete this and the comments below it too